enhancements/authentication/direct-external-oidc-provider.md

```diff
@@ -212,6 +212,168 @@ These changes have already been implemented, and the initial PR for them can be
+
+Managing the CRD will consist of the following:
+
+- When OpenShift oauth server is desired, ensure the `RoleBindingRestriction` CRD exists and matches desired state
+- When OpenShift oauth server is not desired, but `RoleBindingRestriction` resources exist, ensure the `RoleBindingRestriction` CRD exists and matches desired state, set CAO status to `Degraded` with a message stating that `RoleBindingRestriction` resources exist and as such we can not delete the CRD
+- When OpenShift oauth server is not desired, no `RoleBindingRestriction` resources exist, remove the `RoleBindingRestriction` CRD
+
+##### Changes to openshift/api
+
+As the cluster-authentication-operator will now be responsible for the `rolebindingrestrictions.authorization.openshift.io` CRD, it should no longer be added to the openshift/api payload manifests that are included in a payload image and get managed by CVO.
+
+This will likely mean removing the associated files from the hack/update-payload-crd.sh script here: https://github.com/openshift/api/blob/dd0f68969241c0548906ec98c12bb208512cbbb4/hack/update-payload-crds.sh#L6
+
+During an upgrade from a version of OpenShift where the CRD is part of the payload, managed by CVO, to a version of OpenShift where the CRD is managed by the cluster-authentication-operator it is expected to a smooth handover. The anticipated handover flow is:
+
+- CRD is in payload of 4.X, managed by CVO
+- CRD is not in payload of 4.Y, CVO ignores CRD, managed by cluster-authentication-operator
+
+##### Considerations for HyperShift
+
+To ensure commonality in behavior between standalone OpenShift and HyperShift, HyperShift should be updated such that the `HostedControlPlane` controller will act in the same way the cluster-authentication-operator and cluster-kube-apiserver-operator will. Specifically:
+
+- Managing the `RoleBindingRestriction` CRD
+- Setting the appropriate flags on the kube-apiserver when oauth stack is not desired
+- Setting the appropriate admission plugin configurations
+
+This enhancement proposal is scoped only to standalone OpenShift, and while the changes made as part of this EP won't break HyperShift it is recommended that HyperShift is updated in this way. 
+
+##### Considerations for MicroShift
+
+As MicroShift does not run with the OpenShift OAuth stack, it should be updated to ensure that:
+
+- The `RoleBindingRestriction` CRD is not present
+- Setting the appropriate flags on the kube-apiserver when oauth stack is not desired
+- Setting the appropriate admission plugin configurations
+
+This enhancement proposal is scoped only to standalone OpenShift, and while the changes made as part of this EP won't break MicroShift it is recommended that MicroShift is updated in this way. 
+
```

Comment:
Would it make sense to move these into the `### Topology Considerations` section? Or, maybe refer to them there as well?

Reply:
Added some text to the Topology Considerations section to refer to this section.